Core/Utilities/Security/Hashing/HashingHelper.cs

```diff
@@ -20,16 +21,8 @@ public static bool VerifyPasswordHash(string password, byte[] passwordHash, byte
             using (var hmac = new System.Security.Cryptography.HMACSHA512(passwordSalt))
             {
                 var computedHash = hmac.ComputeHash(Encoding.UTF8.GetBytes(password));
-                for (int i = 0; i < computedHash.Length; i++)
-                {
-                    if (computedHash[i]!=passwordHash[i])
-                    {
-                        return false;
-                    }
-                }
+                return computedHash.SequenceEqual(passwordHash) == true ? true : false;
```

Comment:
I think "return computedHash.SequenceEqual(passwordHash);" is enough

Reply:
thanks for helping I fix them